go.mod

```diff
@@ -86,6 +86,298 @@ require (
 	gorm.io/gorm v1.21.15
 )
 
+require (
```

Comment:
@ribasushi is this second `require` not weird?

Reply:
@en0ma no, it is a go1.17+ thing. Note how everything is `indirect`